include/openrave/planner.h

```diff
@@ -491,11 +492,13 @@ class OPENRAVE_API PlannerStatus
     PlannerStatus(const std::string& description, const int statusCode, CollisionReportPtr report=CollisionReportPtr());
     PlannerStatus(const std::string& description, const int statusCode, const IkParameterization& ikparam, CollisionReportPtr report=CollisionReportPtr());
     PlannerStatus(const std::string& description, const int statusCode, const std::vector<dReal>& jointValues, CollisionReportPtr report=CollisionReportPtr());
+    
     virtual ~PlannerStatus();
 
     PlannerStatus& SetErrorOrigin(const std::string& errorOrigin);
     PlannerStatus& SetPlannerParameters(PlannerParametersConstPtr parameters);
 
+    void IncrementCollisionPairCount(CollisionReport& collisionReport); // This assumes that collisionReport plink1 and plink2 fields are populated!
```

Comment:
`IncrementCollisionPairCount` still sounds weird. Maybe `AddCollisionReport`.

Also, parameter should be const since you are not changing it, `const CollisionReport& collisionReport`.